graylog2-web-interface/src/components/indices/IndexSetsComponent.tsx

```diff
@@ -0,0 +1,248 @@
+    color: ${theme.colors.textAlt};
+    font-style: italic;
+  `);
+
+  const statsDisabledText = 'Stats are disabled by default';
+
+  const formatIndexSet = (indexSet: IndexSet) => {
+    const actions = (
+      <ButtonToolbar>
+        <LinkContainer to={Routes.SYSTEM.INDEX_SETS.CONFIGURATION(indexSet.id)}>
+          <Button>Edit</Button>
+        </LinkContainer>
+        <DropdownButton title="More Actions" id={`index-set-dropdown-${indexSet.id}`} pullRight>
+          <MenuItem onSelect={onSetDefault(indexSet)}
+                    disabled={!indexSet.can_be_default || indexSet.default}>Set as default
+          </MenuItem>
+          <MenuItem divider />
+          <MenuItem onSelect={onDelete(indexSet)}>Delete</MenuItem>
+        </DropdownButton>
+      </ButtonToolbar>
+    );
+
+    const content = (
+      <Col md={12}>
+        <IndexSetDetails indexSet={indexSet} />
+
+        <IndexSetDeletionForm ref={
+          (elem) => { formsRef.current = { ...formsRef.current, [`index-set-deletion-form-${indexSet.id}`]: elem }; }
+}
+                              indexSet={indexSet}
+                              onDelete={deleteIndexSet} />
+      </Col>
+    );
+
+    const indexSetTitle = (
+      <Link to={Routes.SYSTEM.INDEX_SETS.SHOW(indexSet.id)}>
+        {indexSet.title}
+      </Link>
+    );
+
+    const isDefault = indexSet.default ? <Label key={`index-set-${indexSet.id}-default-label`} bsStyle="primary">default</Label> : '';
+    const isReadOnly = !indexSet.writable ? <Label key={`index-set-${indexSet.id}-readOnly-label`} bsStyle="info">read only</Label> : '';
+    let { description } = indexSet;
+
+    if (indexSet.default) {
+      description += `${description.endsWith('.') ? '' : '.'} Graylog will use this index set by default.`;
+    }
+
+    let statsString;
+    const stats = indexSetStats[indexSet.id];
+
+    if (stats) {
+      statsString = formatStatsString(stats);
+    }
+
+    return (
+      <EntityListItem key={`index-set-${indexSet.id}`}
+                      title={indexSetTitle}
+                      titleSuffix={<span>{statsEnabled ? statsString : <StatsInfoText>{statsDisabledText}</StatsInfoText>} {isDefault} {isReadOnly}</span>}
+                      description={description}
+                      actions={actions}
+                      contentRow={content} />
+    );
+  };
+
+  const isLoading = !indexSets;
+
+  return (
+    <>
+      <Toolbar>
+        <GlobalStatsCol md={3}>
+          <GlobalStats><strong>Stats for all indices:</strong> {statsEnabled
+            ? formatStatsString(globalIndexSetStats)
+            : <StatsInfoText>{statsDisabledText}</StatsInfoText>}
+          </GlobalStats>
+          <Button onClick={onToggleStats}>{statsEnabled ? 'Disable stats' : 'Enable stats'}</Button>
+        </GlobalStatsCol>
+        <Col md={3}>
+          <SearchForm onSearch={onSearch}
+                      label="Search"
```

Comment:
I think we should keep the search field UI consistent in all the views by removing the `label` and putting the field on the `left side`. For the stats info we can either put it below the search field or in the right side.